gateway-ha/src/test/java/io/trino/gateway/ha/TestGatewayHaSingleBackend.java

```diff
@@ -62,6 +64,26 @@ void setup()
                 "trino1", "http://localhost:" + backendPort, "externalUrl", true, "adhoc", routerPort);
     }
 
+    @ParameterizedTest
+    @MethodSource("protocols")
+    void testEmptyBody(Protocol protocol)
```

Comment:
How does this test relate to `kill_query` procedure change?

Reply:
I must have pulled something in while rebasing without noticing, this was unintentional. I will revert